amanuensis/__init__.py

```diff
@@ -132,13 +132,21 @@ def app_config(
 
 
 def _setup_data_endpoint_and_boto(app):
-    if "AWS_CREDENTIALS" in config and len(config["AWS_CREDENTIALS"]) > 0:
-        #TODO why does it need to be the first one? (use the key value in the object instead of making it a list)
-        value = list(config["AWS_CREDENTIALS"].values())[0]
-        app.boto = BotoManager(value, logger=logger)
-        logger.info("BotoManager initialized")
-    else:
-        logger.warning("Missing credentials for BotoManager, delivery of data will fail.")
+    try:
+        app.s3_boto = BotoManager(
+            config["AWS_CREDENTIALS"]["DATA_DELIVERY_S3_BUCKET"], logger=logger
```

Comment:
The `DATA_DELIVERY_S3_BUCKET` string is actually replaced with the bucket name so this will fail:

https://github.com/chicagopcdc/cloud-automation/blob/e29fb0c6bc3b95f9f23ee7aefdfda9e7297b6eb2/apis_configs/config_helper.py#L140-L144